Shippo/Track.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Shippo {
+    [JsonObject (MemberSerialization.OptIn)]
+    public class Track : ShippoId {
+        [JsonProperty (PropertyName = "carrier")]
+        public object Carrier { get; set; }
```

Comment:
Just tried changing them to private, it doesn't allow me to access the properties. Instead I just removed the getters and setters and tests pass 👍